# Author: Jiri Furst <jiri.furst@gmail.com>
# Boost dll::import was renamed to dll::import_symbol  
# see https://github.com/boostorg/dll/commit/6b2e3c7f660199f2bb0d69b667a38ee21cd49626

--- dakota-6.16.0-public-src-cli/src/PluginInterface.cpp.orig	2022-07-20 11:26:50.680437606 +0200
+++ dakota-6.16.0-public-src-cli/src/PluginInterface.cpp	2022-07-20 11:27:59.736433721 +0200
@@ -111,7 +111,7 @@
 {
   if (pluginInterface) return;
   try {
-    pluginInterface = boost::dll::import<DakotaPlugins::DakotaInterfaceAPI>
+    pluginInterface = boost::dll::import_symbol<DakotaPlugins::DakotaInterfaceAPI>
       (pluginPath,
 	 "dakota_interface_plugin"  // name of the symbol to import
 	 // TODO: append .dll, .so, .dylib via
